setup.py

```diff
@@ -37,7 +37,7 @@
         "Programming Language :: Python :: 3.10",
     ],
     packages=find_packages(),
-    python_requires=">=3.10, <3.11",
+    python_requires=">=3.10, <3.12",
```

Comment:
Does py3.11 just work? We should probably also add testing to our CI matrix before enabling this change. wdyt?

Reply:
Yeah, it just works , can leave it out though.  Let me revert

Reply:
haha Ayush you barely beat me to it

Reply:
Reverted